lib/components/output-area.js

```diff
@@ -27,12 +28,24 @@ const OutputArea = observer(({ store: { kernel } }: { store: store }) => {
     }
   }
 
+  const handleClick = () => {
+    if (!kernel || !kernel.outputStore) return;
+    //convert output to a plain js object
+    const output = toJS(kernel.outputStore.outputs[kernel.outputStore.index]);
```

Comment:
I'm not sure if we really need to convert it but it doesn't hurt.

Reply:
I had some error without it (not consistently)